airflow/configuration.py

```diff
@@ -50,7 +50,7 @@
 # show Airflow's deprecation warnings
 if not sys.warnoptions:
     warnings.filterwarnings(action='default', category=DeprecationWarning, module='airflow')
-    warnings.filterwarnings(action='default', category=PendingDeprecationWarning, module='airflow')
+    warnings.filterwarnings(action='default', category=RemoveInAirflow3DeprecationWarning, module='airflow')
```

Comment:
I'm not sure if we should leave `PendingDeprecationWarning` as is and just add `RemoveInAirflow3DeprecationWarning` rather than replace?

Reply:
I’d probably just keep this as-is. Also note that adding `RemoveInAirflow3DeprecationWarning` here does not actually do anything, since it is a subclass of `DeprecationWarning` and already enabled by line 52.

Reply:
Yes I was wondering about:
```diff
-warnings.filterwarnings(action='default', category=DeprecationWarning, module='airflow')
-warnings.filterwarnings(action='default', category=PendingDeprecationWarning, module='airflow')
+warnings.filterwarnings(action='default', category=RemoveInAirflow3DeprecationWarning, module='airflow')
```
our goal here is not to have general `DeprecationWarning` in the code base

Reply:
https://github.com/apache/airflow/pull/25799/commits/a54ee211784de62ba3b880b3f6f4b969e8880ac4